azure-pipelines.yml

```diff
@@ -34,17 +34,10 @@ jobs:
   pool:
     vmImage: 'macOS-10.14'
   steps:
-  - task: UseDotNet@2
-    displayName: 'Use .NET Core SDK 3.1.101'
+  - task: DotNetCoreInstaller@0
```

Comment:
What's the difference between the `UseDotNet` and `DotNetCoreInstaller` tasks?

Reply:
Well according to [DotNetCoreInstallerV0](https://github.com/microsoft/azure-pipelines-tasks/tree/master/Tasks/DotNetCoreInstallerV0) and [UseDotNetV2](https://github.com/microsoft/azure-pipelines-tasks/tree/master/Tasks/UseDotNetV2) readme files, the latter seems to be a newer version of the former. Planning to change the task back to `UseDotNet@2`, thanks for reviewing this!